### PR TITLE
Support principal ids for e2e

### PR DESCRIPTION
**Pull request by eduwercamacaro**

### CODE REVIEW

File: server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java [hunk 1]
```diff
@@ -140,7 +140,12 @@ private ProcessorExecutionContext buildExecutionContext(Record<String, Command>
     }
 
     public void onPartitionClaimed() {
+        if (partitionIsClaimed) {
+            throw new RuntimeException("Re-claiming partition! Yikes!");
```
Comment (coltmcnealy-lh) [inline server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java:144]: This looks like I wrote this code :joy: You are picking up my style!

File: server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java [hunk 2]
```diff
@@ -155,11 +160,6 @@ public void onPartitionClaimed() {
     private void rehydrateTenant(TenantModel tenant) {
         TenantScopedStore coreDefaultStore =
                 TenantScopedStore.newInstance(this.nativeStore, tenant.getId(), new BackgroundContext());
-        if (partitionIsClaimed) {
-            throw new RuntimeException("Re-claiming partition! Yikes!");
```
Comment (coltmcnealy-lh) [inline server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java:159]: Oh never mind, I think you just copied it from me haha

File: server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java (hunk 1 above)
Comment (eduwercamacaro) [reply server/src/main/java/io/littlehorse/server/streams/topology/core/processors/CommandProcessor.java:144]: That's your code. just moving

**Review by coltmcnealy-lh — APPROVED** [review]
Looks good. But what about checking to make sure we don't doubly-hydrate the `default` tenant if it exists in the store?

**Comment by eduwercamacaro** [comment]
I need to a add to prevent double task rehydration for default tenant.

**Pull request merged**